python/dgl/backend/pytorch/tensor.py

```diff
@@ -14,8 +14,8 @@
 from ...base import dgl_warning
 
 if LooseVersion(th.__version__) < LooseVersion("1.5.0"):
-    dgl_warning("Detected an old version of PyTorch. Suggest using torch>=1.5.0 "
-                "for the best experience.")
+    raise Exception("Detected an old version of PyTorch. Please update torch>=1.5.0 "
+                    "for the best experience.")
```

Comment:
Have you tested if this works for nightly build?

Reply:
It can handle the nightly version. 
```
LooseVersion("1.7.1dev20210119") > LooseVersion("1.5") # True
LooseVersion("1.7.1dev20210119") > LooseVersion("1.7") # True
LooseVersion("1.7.1dev20210119") > LooseVersion("1.7.1") # True
LooseVersion("1.7.1dev20210119") > LooseVersion("1.8") # False
```